docs/reference/commandline/daemon.md

```diff
@@ -67,6 +67,7 @@ weight = -1
       --tlsverify                            Use TLS and verify the remote
       --userns-remap="default"               Enable user namespace remapping
       --userland-proxy=true                  Use userland proxy for loopback traffic
+      --userland-proxy-bin=""                Use this userland proxy binary, if userland-proxy is set
```

Comment:
It's used as a `Cmd.Path` (i.e. an argv[0]) so perhaps it should be `--userland-proxy-path`?